### ISSUE TITLE
Cannot import fasttrackml  package

### ISSUE THREAD

**Issue by elehcimd** [G-Research/fasttrackml#956]
I cannot import the package. The system is MacOS Monterey, M2 2022.

Instructions to reproduce the issue:

```
> pip install fasttrackml
Collecting fasttrackml
  Obtaining dependency information for fasttrackml from https://files.pythonhosted.org/packages/e9/a6/0884b628f744299f31be118ee95e3811cf2af92e607176bcf462395bb18d/FastTrackML-0.4.0-py3-none-macosx_11_0_arm64.whl.metadata
  Downloading FastTrackML-0.4.0-py3-none-macosx_11_0_arm64.whl.metadata (19 kB)
Downloading FastTrackML-0.4.0-py3-none-macosx_11_0_arm64.whl (22.3 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 22.3/22.3 MB 4.2 MB/s eta 0:00:00
Installing collected packages: fasttrackml
Successfully installed fasttrackml-0.4.0

[notice] A new release of pip is available: 23.2.1 -> 24.0
[notice] To update, run: python3.11 -m pip install --upgrade pip
> python
Python 3.11.6 (main, Nov 28 2023, 09:22:32) [Clang 14.0.0 (clang-1400.0.29.202)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import fasttrackml
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'fasttrackml'
>>>
```

**Comment by elehcimd** [comment]
The issue can be reproduced also in a clean Google Colab,

```
!pip install fasttrackml
import fasttrackml
```

**Comment by jgiannuzzi** [comment]
Hey @elehcimd, have a look at #905, specifically this comment: https://github.com/G-Research/fasttrackml/issues/905#issuecomment-1952831140.

tl;dr you can either `import mlflow` or `pip install fasttrackml==0.5.0b2` if you need metric contexts.
